subsys/bootloader/Kconfig

```diff
@@ -120,16 +120,18 @@ if IS_SECURE_BOOTLOADER
 
 config PM_PARTITION_SIZE_PROVISION
 	hex
-	default 0x0 if SOC_NRF9160
-	default 0x1000 if !SOC_NRF9160
-	prompt "Flash space reserved for PROVISION" if !SOC_NRF9160
+	default 0x0 if SOC_NRF9160 || SOC_NRF5340_CPUAPP
+	default 0x800 if SOC_NRF5340_CPUNET
```

Comment:
Could we use the DT symbol here?

Reply:
I changed it to use FPROTECT_BLOCK_SIZE.